python/desc/sims/GCRCatSimInterface/DatabaseEmulator.py

```diff
@@ -172,6 +172,8 @@ class DESCQAObject(object):
 
     objectTypeId = None
     verbose = False
+    tableid = 'galaxy'
```

Comment:
Why did you need to add these?

I *thought* I had implement the `CompoundDESCQAInstanceCatalog` class so that it evaluated the equality of `DESCQAObject`s on things that were relevant to `DESCQAObjects`.  Was something failing without these member variables?

Reply:
I don't think I need those. I think I was trying to solve something in the compound Instance catalog and this was an attempted solution I didn't remove. I can't take those out now.